src/frontends/paddle/tests/requirements.txt

```diff
@@ -1,5 +1,5 @@
 # PaddlePaddle - generate test models
-paddlepaddle==2.4.1
+paddlepaddle==2.5.0
```

Comment:
I'm not sure that we can safely change PDPD and ONNX deendencies, because some updated in product part are required as well.
E.g. PDPD 2.5.x is not even part of current master. Regarding ONNX - we need to port whole PR which enables ONNX 1.13 migration

Reply:
Thanks, reverted, let's wait support on master first